HELPS/ClientApp/src/components/adminWorkshops/adminWorkshops.tsx

```diff
@@ -0,0 +1,104 @@
+import {
+    Collapse,
+    Container,
+    Navbar,
+    NavbarBrand,
+    NavbarToggler,
+    NavItem,
+    NavLink,
+    Form,
+    FormGroup,
+    Label,
+    Input,
+    Button
+} from 'reactstrap';
+import {AppState} from '../../types/store/StoreTypes';
+import {Dispatch} from 'redux';
+import {connect} from 'react-redux';
+import {Link} from 'react-router-dom';
+
+import * as React from 'react';
+import {Component} from 'react';
+import {
+    WorkshopEvent,
+    AdminWorkshopDispatchProps,
+    AdminWorkshopProps,
+    AdminWorkshopStateProps
+} from '../../types/components/AdminWorkshopsTypes';
+import {
+    addSkill
+} from '../../store/actions/AdminWorkshopActions';
+
+
+class AdminWorkshops extends Component{
+
+    componentDidMount(): void {
+        //this.props.retrieveWorkshops();
+    }
+
+    addSkillSet = (skill) => {
+       
+        adminWorkshops.dispatch({type: 'ADD_SKILL'
+                
+            });     
+    }
+
+    setWorkshops = () => {
+        //pass in skill name
+    }
+
+    isCurrent = () => {
+        if(this.isCurrent === true){
+            return( 
+                <text>CURRENT!!!</text>
+            )
+        }
+    }
+    render() {
+       
+        return (
```

Comment:
You are on the right track here, have a look at the mock (in the group chat, open using Adobe XD) to give you a bit more of a tangible feel for how the final product should look.

You need to add the same calendar component we worked on in the student side and add a getWorkshops action to your reducer